src/mediatype-icon.ts

```diff
@@ -10,13 +10,17 @@ import { textsIcon } from './assets/img/icons/mediatype/texts';
 import { tvIcon } from './assets/img/icons/mediatype/tv';
 import { videoIcon } from './assets/img/icons/mediatype/video';
 import { webIcon } from './assets/img/icons/mediatype/web';
+import { collectionIcon } from './assets/img/icons/mediatype/collection';
 
 @customElement('mediatype-icon')
 export class MediatypeIcon extends LitElement {
   @property({ type: String }) mediatype = '';
 
+  @property({ type: String }) icontype = 'tile';
```

Comment:
For things like this that have limited options, add a `type` to restrict the options, ie:
```
type IconType = 'list' | 'tile';
...
@property({ type: String }) icontype: IconType = 'tile';
```
This will give you type checking on it.

---

I also have another suggestion below about this property.

Reply:
used boolean